re-structures literature and adds analysis section

Nikoleta-v3 opened the pull request:
This pull request fixes the structure of the paper. The structure has been discussed and it has been 
agreed upon. 

Moreover, this pull request is mainly so @drvinceknight can review the analysis section.

Nikoleta-v3 committed:
first re-structed version.

Brutally moved things around. Need to be
re read and cleaned

Nikoleta-v3 committed:
cleaning after restructuring.

Reading through and filling the gaps.
Still stuff to be added.

I had to rename several section and though it's not
following exaclty a timeline is closer to what we have discussed.

Added information to axelrod's work and more stuff on:
- the noise section
- the early experiments

Nikoleta-v3 committed:
initial write up for the analysis.

Nikoleta-v3 committed:
generate updated timeplot with points

+ minor corrections

Nikoleta-v3 committed:
New structure

- Restructured the paper ones again. Now very simple going throw highlight events that took place in
a chronological order. The current sections (of the timeline) are:
  - Origin and (1961-1972)
  - Axelrod’s Tournaments (1981-1984)
  - Further work and Criticism on Computer Tournaments (1984-1993)
  - Evolutionary Dynamics (1987-1999)
  - Modern Approaches (1995-2011)
  - Zero Determinant (2012 - 2015)
  - Current Area (2015 - 2017)

+ Some of the analysis cause I forgot to commit :smile:

Cleaning notebooks.

The code for retrieving data from Axelbib and pushing
articles to Axelbib have been removed from notebooks
and are now scripts.

Also all the code for cleaning the data were brought together
on a notebook (A.0.1. Cleaning data)

Write the analysis

Cleaning the main body of the article

This includes reading through and correcting
the language. As well as adding connecting sentences.

Clear analysis.

Few things left to do. Add 2-3 final sentences and
generate the network plots with legends.

Nikoleta-v3 committed:
morning tweaks

Fixing the network plots. This includes making
legends and added a few ending sentences.

Nikoleta-v3 committed:
first write up of the analysis.

This is the first write up of the authorship analysis.
This includes new stuff such as the comparison between
the prisoner's dilemma and two other topics of game theory.
These topics are,

- price of anarchy
- auction games

This commit is up to the comparison section.

Nikoleta-v3 committed:
creates an environment.yml for the literature

Nikoleta-v3 committed:
write up of subgraphs section.

Nikoleta-v3 committed:
read through and write conclusion

Nikoleta-v3 committed:
clean conclusion

drvinceknight reviewed (changes requested):
I will continue more tomorrow and finish by Monday. Looks good so far @Nikoleta-v3.

Nikoleta-v3 committed:
clean and structure nodebooks

Nikoleta-v3 committed:
analysis outputs from write up.

Output tables, images and numbers to input
in the main file.

Nikoleta-v3 committed:
Address comments on the introduction.

Includes,

- making the prisoners dilemma part of the introduction
- adding about Axelrod's book
- fixing legend in timeplot
- changing title Current area to Contemporary period

Nikoleta-v3 committed:
Address comments on Origin and Primal research

Nikoleta-v3 committed:
Address comments on Axelrod' section.

Left to check the natural selection tournament and write down the maths

Nikoleta-v3 committed:
addressed comments on ecological tournament

Nikoleta-v3 committed:
address comments on Response to the ct

Nikoleta-v3 committed:
change ecologica image

Nikoleta-v3 committed:
addressed comments on evolutionary dynamics.

Nikoleta-v3 committed:
re-writing modern era

Nikoleta-v3 committed:
pair writing analysis introduction.

drvinceknight committed:
Add skeleton and suggestions for data description.

drvinceknight committed:
Add numerous TODOs about co author net.

Nikoleta-v3 committed:
Analysis chapter (#19)

* Re writting analysis.

After a meeting with Vince the structure of the analysis
was determined #15. Currently writing all the analysis.
The following sections has been complinted:

- Data collection
- Pre Analysis
- Contructing the networks
- Collaborative behaviour

Left to do is:

- Influence, which is mainly written just missing the conclusions
- Conclusion

* running notebooks for analysis

* tex files.

For variables and tables

* Last subsection and conclusions

* clears notebooks.

All notebooks 1-8 run. All tables and constants
are manually written to file

* added notes to what needs to be re written.

* remove keywords that where not used.

A list of keywords have been used to collected data
on the prisoner's dilemma. But actually a few of them
did not return a single article.

These words were removed from the table. The
check was done in the notebook 'Extra Check keywords'

* additions to text

This includes:

- a sentence about the number of keywords for the data collection
- sources citation
- example for the levenshtein distance
- Definition of co authorship network

* first re-write without trends are conclusion

* Finish analysis.

Only conclusion left.

* clean tex files

The files autogenerated for the tables and the variables.

* clean images

Removed unwanted plots and commit new ones.

* run notebooks & added time series analysis

* sty file

* add references

* conclusion

* clean the literature
